woof-code/support/petbuilds.sh

```diff
@@ -200,25 +200,82 @@ for i in ../rootfs-petbuilds/busybox ../rootfs-petbuilds/*; do
         fi
 
         rm -rf ../petbuild-output/${NAME}-${HASH}/root/.ccache
+        rm -rf ../petbuild-output/${NAME}-${HASH}/root/.cache/*
+        rm -rf ../petbuild-output/${NAME}-${HASH}/root/.dbus/*
+        rm -rf ../petbuild-output/${NAME}-${HASH}/root/.thumbnails/*
+        
         rm -rf ../petbuild-output/${NAME}-${HASH}/tmp
         rm -rf ../petbuild-output/${NAME}-${HASH}/etc/ssl
         rm -f ../petbuild-output/${NAME}-${HASH}/etc/resolv.conf
         rm -f ../petbuild-output/${NAME}-${HASH}/root/.wget-hsts
 
-        rm -rf ../petbuild-output/${NAME}-${HASH}/usr/share/man
-        rm -rf ../petbuild-output/${NAME}-${HASH}/usr/share/info
-        rm -f ../petbuild-output/${NAME}-${HASH}/usr/share/icons/hicolor/icon-theme.cache
         rm -rf ../petbuild-output/${NAME}-${HASH}/lib/pkgconfig
-        rm -rf ../petbuild-output/${NAME}-${HASH}/usr/lib/pkgconfig
-        rm -rf ../petbuild-output/${NAME}-${HASH}/usr/share/pkgconfig
-        rm -rf ../petbuild-output/${NAME}-${HASH}/usr/include
-
+        
+        for usrfld2 in usr usr/local
+        do
```

Comment:
Is there any specific file or directory that's added by any existing petbuild and needs to be deleted, or these changes are supposed to handle any existing and future leak of extra files? I mean, is this big change supposed to solve a concrete, existing problem, or protect against future problems?

Reply:
Protect against future problems and improve flexibility

Reply:
I tried very hard to keep petbuilds.sh as small as possible, because the collection of 3builddistro petbuilds should remain small if we want to be able to maintain it (it's just Puppy-specific applications, traditional Puppy stuff like Geany with Puppy customization, and Puppy applications we want at a specific version, like JWM).

The whole idea behind #1984 is to make it possible to build a Puppy from other distro libraries, plus a traditional Puppy desktop with a basic set of applications, built from source. Building 'core' stuff like glibc, OpenSSL, CUPS or PulseAudio, or the machinery to support that, doesn't belong here. IMHO, it only adds complexity that may hide real problems, and why change something that achieves its goal?

I'm not obsessed with petbuilds.sh and trying to protect it against changes or something, only trying to keep it short and readable, at least while it's still new and buggy here and there.

Reply:
However after compiling the program and installed with make install. Sometimes it leaves residual files which adds weight to the petbuild output. So additional cleanup to newly compiled petbuild output is needed.

Reply:
The question is, does petbuild.sh need to change so much. Is there any specific file or directory that's not cleaned up today that needs to be dealt with? If yes, petbuilds.sh should delete that, and only that. Otherwise, we should look at builtin_files, see what needs to be cleaned up, and deal with that.

Reply:
However its hard to deal with builtin_files at this moment due to merging of packages which supposed not to be  happen